src/test/resources/application-test.yaml

```
@@ -0,0 +1,28 @@
+spring:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)